deps/v8/gypfiles/toolchain.gypi

```diff
@@ -61,7 +61,7 @@
     'v8_android_log_stdout%': 0,
 
     # Force disable libstdc++ debug mode.
-    'disable_glibcxx_debug%': 0,
+    'disable_glibcxx_debug%': 1,
```

Comment:
You should be able to override it from common.gypi but I don't expect it's necessary to do so.

Checked STL containers have different name mangling in libstdc++.  I think the worst case is that the add-on fails to load at runtime, but not that it silently does the wrong thing.